regression-tests/backends/geoip-master

```diff
@@ -51,11 +51,17 @@ domains:
       - a: $geoipregionip
     unknown.service.geo.example.com:
       - a: 127.0.0.1
+    earth.map.geo.example.com:
+      - a: $geoipregionip
```

Comment:
I don't see the point of returning `$geoipregionip`, please explain.
If you did it without real motivation, I think you could hardcode a dedicated value. it would allow you to have the expected result as a simple file within the regression test folder

Reply:
Moved to TXT